app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -26,37 +27,51 @@ public Item(int id, boolean deleted, String type, String by, long time, String t
         this.dead = dead;
     }
 
-    /** The item's unique ID */
+    /**
+     * The item's unique ID
```

Comment:
Yeah, generally getter/setter doesn't need a comment.

The only useful comment we can make is public API/interface's javadoc.
It makes implementer's life so much easier.